add zendesk retention policy info (#36615)
DataDog/documentation: content/en/account_management/guide/manage-your-support-tickets.md

<p>If your browser has ad-blockers, turn them off to see if this allows you to sign in. Some ad-blockers have their own list of exceptions. In this case, add <strong>datadog.zendesk.com</strong> to the allow list.</p>

<h3>Ticket is no longer available</h3>

> <p>Closed tickets, including their attachments, are deleted 15 months after their last update.</p>

Reviewer: ```suggestion
<p>Datadog deletes closed tickets, including their attachments, 15 months after their last update.</p>
```
I think it helps to identify who's doing what if we can, instead of using passive voice

> <p>Tickets are automatically removed after the retention period ends. Closed tickets are deleted 15 months after their last update, and attachments 12 months.</p>

Reviewer 2: ```suggestion
<p>Tickets are automatically removed after the retention period ends. Closed tickets are deleted 15 months after their last update, and attachments are deleted after 12 months.</p>
```

Author: They decided to update the retention policy a little bit anyway, which automatically fixes this clarity issue.

> <p>If you need help with a related issue, you can open a new ticket or search our documentation.</p>

Reviewer: ```suggestion
<p>If you need help with a related issue, you can open a new ticket or search the Datadog documentation.</p>
```
"our" can be a little confusing when we're talking about both our product and an integration with a different service, so I find some specificity helpful!
